vars/main.yml

```diff
@@ -431,6 +431,35 @@ pgbackrest_cron: []
 #  - { name: "Full Backup", type: "full", hour: "06", minute: "30", dow: "0" }
 #  - { name: "Diff Backup", type: "diff", hour: "06", minute: "30", dow: "1-6" }
 
+cron_jobs: []
+#  - name: "PgBackerest: Full Backup"
+#    file: /etc/cron.d/pgbackrest
+#    user: "{{ patroni_superuser_username }}"
+#    minute: "30"
+#    hour: "6"
+#    day: "*"
+#    month: "*"
+#    weekday: "0"
+#    job: "/usr/bin/pgbackrest --type=full --stanza={{ pgbackrest_stanza }} backup"
+#  - name: "PgBackerest: Diff Backup"
+#    file: /etc/cron.d/pgbackrest
+#    user: "{{ patroni_superuser_username }}"
+#    minute: "30"
+#    hour: "6"
+#    day: "*"
+#    month: "*"
+#    weekday: "1-6"
+#    job: "/usr/bin/pgbackrest --type=diff --stanza={{ pgbackrest_stanza }} backup"
```

Comment:
```suggestion
#    job: "[ "$(psql -tAXc 'select pg_is_in_recovery()')" = "f" ]; then pgbackrest --type=diff --stanza={{ pgbackrest_stanza }} backup; fi"
```

run the pgbackrest process on the Primary server (instead of each server in the cluster).